addons/website/static/src/snippets/s_searchbar/000.js

```diff
@@ -251,6 +251,11 @@ publicWidget.registry.searchBar = publicWidget.Widget.extend({
      * @private
      */
     _onFocusOut: function () {
+        // If focus is away from the search modal, hide it.
+        const modal = Modal.getOrCreateInstance(this.$input.closest('#o_search_modal'));
+        if (modal) {
+            modal.hide()
+        }
```

Comment:
Wasn't the `css_editable_mode_hidden` xml solution I [explained](https://github.com/odoo/odoo/pull/164129#pullrequestreview-2035205461) working?

Reply:
No it is not working

Reply:
What is not working? I had a quick look on runbot, and by adding `css_editable_mode_hidden` on the `#o_search_modal` in the DOM, then entering edit mode, it is hidden (as if it was not opened in the first place)?
What issue did you face